package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "rxjs-observed-decorator",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
if you're going to manually bump the version, this needs value to change in `package-lock.json` as well.

Reply:
Done!